### PR TITLE
Abnormal lock file needs to be cleared

### PR DESCRIPTION
**Pull request by ahuigo**
For https://github.com/lino-levan/astral/issues/117

If the download of chrome is interrupted unexpectedly, lock file will be retained forever.

We need to clean abnormal lock file

### CODE REVIEW

File: src/cache.ts [hunk 1]
```diff
@@ -301,6 +301,24 @@ class Lock {
...
+  /** Clean expired lock path */
+  checkExpiredLockPath() {
+    // if this.path's create time is older than cacheTTL, remove it
+    try {
+      const fileInfo = Deno.statSync(this.path);
+      const lockTTL = 300 * 1000;
+      if (fileInfo.birthtime && Date.now() - fileInfo.birthtime.getTime() < lockTTL) {
+        // Deno.removeSync(this.path);
+        console.warn("%c There is an old lock file(%s), you should remove it.", 'color: #ff0000', this.path);
+      }
```
Comment (lino-levan) [inline src/cache.ts:316]: I wonder if we should just remove it. If not, this behavior is fine in CI but not fine for a regular user to encounter. If we're not in CI we should prompt the user with "there is an old lock file, this is probably due to a failed download, do you want to remove it? [Y/n]" or something like that

Comment (ahuigo) [reply src/cache.ts]: I prefer to just remove it. Simple for user and fine for CI.
```
Deno.removeSync(this.path);
console.log(`%c There is an old lock file (${this.path}), this is probably due to a failed download. It has been removed automatically.`, 'color: #ff0000');
```

Prompt is not fine for CI as you said.

        const userInput = prompt(`\x1b[31m There is an old lock file (${this.path}), this is probably due to a failed download. Do you want to remove it? [Y/n]:\x1b[0m`);
        if (userInput && userInput.toLowerCase() === 'y') {
          Deno.removeSync(this.path);
          console.log(`Old lock file ${this.path} has been removed.`);
        } else {
          console.log(`Lock file ${this.path} has not been removed.`);
        }

**Review by lino-levan — APPROVED** [review]
If you fix formatting this looks good to me. Thanks.

**Comment by lino-levan** [comment]
This is failing a test, any ideas?

**Comment by ahuigo** [comment]
I made a fully test, I found that I make a stupid mistake, I wrote `>ttl` to `<ttl`.

Unit test is passed so far.

```
ok | 57 passed | 0 failed (4m29s)
```

**Comment by lino-levan** [comment]
Thank you for the contribution!

**Pull request merged**